cypress/integration/word-count.spec.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+describe('Test word count with', () => {
+  beforeEach(() => {
+    cy.visitTestEditor()
+  })
+
+  it('empty note', () => {
+    cy.codemirrorFill('')
+    cy.wait(500)
+    cy.get('button[data-cy="sidebar-btn-document-info"]').click()
+    cy.get('[data-cy="document-info-modal"]').should('be.visible')
+    cy.get('b[data-cy="document-info-word-count"]').should('have.text', '0')
```

Comment:
you don't need that b. The tag selector should be enough.

Reply:
fixed